auth/login.html

```diff
@@ -0,0 +1,111 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Login - Wrixby ECom || An E-Commerce Store</title>
+    <script src="https://kit.fontawesome.com/303bcab846.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="/assets/css/style.css">
+</head>
+
+<body>
+    <header class="navbar">
+        <nav class="nav">
+            <div class="navbar-brand">
+                <a href="/" class="navbar-logo">
+                    <h3 class="text-primary">Wrixby ECom</h3>
+                </a>
+            </div>
+            <ul class="navbar-menu">
+                <li class="nav-link">
+                    <a href="/products/">
+                        Products
+                    </a>
+                </li>
+                <li class="nav-link">
+                    <a href="/auth/login.html">
+                        <button class="button button-primary button-round">
+                            Login
+                        </button>
+                    </a>
+                </li>
+                <li class="nav-link">
+                    <a href="/wishlist/">
+                        <button class="button button-info-outline button-floating">
+                            <i class="fa-solid fa-heart"></i>
+                        </button>
+                    </a>
+                </li>
+                <li class="nav-link">
+                    <a href="/cart/">
+                        <button class="button button-warning button-floating">
+                            <i class="fa-solid fa-cart-plus"></i>
+                        </button>
+                    </a>
+                </li>
+            </ul>
+        </nav>
+    </header>
+    <main>
+        <div class="container">
+            <section class="listing-container margin-vertical-5 border-radius-sm margin-horizontal-auto">
+                <div class="card card-vertical border-radius-md">
+                    <div class="card-header text-center">
+                        <h3>Login</h3>
+                    </div>
+                    <div class="card-body">
+                        <form action="javascrpt:;" class="form">
+                            <div class="margin-bottom-3">
+                                <label for="TxtEmail" class="form-label">
+                                    <strong>Email Address</strong>
+                                </label>
+                                <input type="text" name="" placeholder="Enter Your Email ID" id="TxtEmail"
```

Comment:
Suggestion: Dont use Pascal case in HTML, it will get the job done but its just better to use lowercase with hyphens in between, in HTML.

Reply:
Thanks! Will keep that in mind.